src/test/java/hw2/base/Const.java

```diff
@@ -0,0 +1,40 @@
+package hw2.base;
+
+public class Const {
+
+    //names or titles
+    public static final String titleExpected = "Home Page";
+    public static final String login = "Roman";
+    public static final String password = "Jdi1234";
+    public static final String userName = "ROMAN IOVLEV";
+    public static final String menuButtons = "HOME\n"
+            + "CONTACT FORM\n"
+            + "SERVICE\n"
+            + "METALS & COLORS";
+    public static final String menuButtonsOnTheLeft = "Home\n"
+            + "Contact form\n"
+            + "Service\n"
+            + "Metals & Colors\n"
+            + "Elements packs";
+
+    //locators
+    public static final String userIcon = "profile-photo";
```

Comment:
Absolutely not. It's not correct.

Imagine that you have 20+ pages. How much public static final String you will have?
How bigger will be class?


In task you have to use only driver. So they  correct way is:

```
    protected void performLogin() {
        driver.findElement(By.className("profile-photo")).click();
        driver.findElement(By.id("name")).sendKeys(login);
        driver.findElement(By.cssSelector("#password")).sendKeys(password);
        driver.findElement(By.id("login-button")).click();
    }
```